### ISSUE TITLE
Windows Online Build fails with 'Illegal characters in path'

### ISSUE THREAD

**Issue by shomeier** [eclipsesource/tabris-js#1368]
I just want to experiment a bit with a Windows 10 build but the Online Build fails with the following exception:

> [00:04:15] C:\Program Files (x86)\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(4136,5): error MSB3021: Unable to copy file "C:\projects\tabris-js-build-template\build\platforms\windows\www\**" to "bin\x86\Release\www\**". Illegal characters in path. [C:\projects\tabris-js-build-template\build\platforms\windows\EclipseSource.Tabris.CordovaLauncher.csproj]

Attached is the full detailed log.
tia
[Windows_Build_Tabris.txt](https://github.com/eclipsesource/tabris-js/files/1054933/Windows_Build_Tabris.txt)

**Comment by tbuschto** [comment]
It works on our test project. Are there any special characters (not a-z, A-Z, 0-9, dash, dot or underscore) on your project name, module names, any other files/directories?

**Comment by shomeier** [comment]
Yes, I had @types/node on my devDependencies which result in an equal named folder. I removed it but still getting the error. Will further investigate and try to copy my root project folder to a Windows Filesystem to see if there are other conflicts.

**Comment by shomeier** [comment]
I now zipped my complete project folder and unzipped it on a Windows 10 VM. I am not sure if there is some automatic path conversion during this process. While unzipping it on Windows 10 there have been complaints that some paths are too long but not about invalid characters.

**Comment by tbuschto** [comment]
There's a good chance that it's the path length. See https://stackoverflow.com/questions/34247366/msbuild-cannot-recursively-copy-files-with-copy-task. If the path that are too long are part of node_modules we would have to look at the offending module. Otherwise you have to change your sub directory names to be shorter or flatter. Please let us know if you manage to fix it.

**Comment by shomeier** [comment]
There were many paths problems but I think most if not all were in build/cordova/node_modules.
I can remember that there was a problem with some test files in build/cordova/node_modules/cordova-plugin-camera.
But as I said above I just wanted to experiment a bit with Windows 10 so it is not really important for me at the moment ;)

**Comment by shomeier** [comment]
With latest RC-2 the online build just passed ;)

**Comment by shomeier** [comment]
Closed too early: Build passed but when trying to download appx file I get an "Access Denied" from Amazon S3
